test/extended/machines/scale.go

```diff
@@ -186,12 +188,31 @@ func scaleMachineSet(name string, replicas int) error {
 	return nil
 }
 
+func getOperatorsNotProgressing(c configclient.Interface) map[string]metav1.Time {
+	operators, err := c.ConfigV1().ClusterOperators().List(context.Background(), metav1.ListOptions{})
+	o.Expect(err).NotTo(o.HaveOccurred())
+	result := map[string]metav1.Time{}
+	for _, operator := range operators.Items {
+		if operator.Name == "machine-config" {
```

Comment:
We don't need this exception, do we?  I don't think we expect any ClusterOperator to go `Progressing=True` just based on Machine scaling, even the MCO.

Reply:
OhOh. Gotcha.